src/bundle/Resources/config/services/suggestions.yaml

```diff
@@ -0,0 +1,36 @@
+parameters:
+    ibexa.site_access.config.default.search.suggestion.min_query_length: 3
+    ibexa.site_access.config.default.search.suggestion.result_limit: 5
+
+services:
+    _defaults:
+        autoconfigure: true
+        autowire: true
+        public: false
+
+    Ibexa\Bundle\Search\Controller\SuggestionController:
+        tags:
+            - { name: 'container.service_subscriber', key: 'serializer', id: 'ibexa.search.suggestion.serializer' }
+
+    Ibexa\Bundle\Search\ArgumentResolver\SuggestionQueryArgumentResolver:
+        arguments:
+            $defaultLimit: '%ibexa.site_access.config.default.search.suggestion.result_limit%'
```

Comment:
This should be replaced with `ConfigResolver` instead.

When the container is built, site access is undefined, so this value will always represent the default value for the system. To use the right configuration value, including site access settings, this parameter has to be acquired at runtime.